brightchain-lib/src/lib/access/checksum.ts

```
@@ -0,0 +1,38 @@
+  public static validateChecksum(checksum: ChecksumBuffer): boolean {
+    return this.getService().validateChecksum(checksum);
+  }
+
+  public static validateDataChecksum(
+    data: Buffer,
+    checksum: ChecksumBuffer,
+  ): boolean {
+    const calculatedChecksum = this.calculateChecksum(data);
+    return this.compareChecksums(calculatedChecksum, checksum);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)